requirements.txt

```diff
@@ -1,6 +1,7 @@
 requests>=2.0
 oauthlib
-requests-oauthlib>=0.6
+# requests-oauthlib>=0.6.2
+git+https://github.com/singingwolfboy/requests-oauthlib.git@oauth2-request-accept-client-id-and-secret#egg=requests_oauthlib
```

Comment:
Can't merge this pull request until https://github.com/requests/requests-oauthlib/pull/251 is merged and released.